docs/api/alert.md

```diff
@@ -1,8 +1,6 @@
 ---
 title: "ion-alert"
 hide_table_of_contents: true
```

Comment:
```suggestion
```

Reply:
Can also remove the `<h2 className="table-of-contents__title">Contents</h2>` below too